nexus/internal-api/src/lib.rs

```diff
@@ -554,6 +554,25 @@ pub trait NexusInternalApi {
         rqctx: RequestContext<Self::Context>,
         policy: TypedBody<ClickhousePolicy>,
     ) -> Result<HttpResponseUpdatedNoContent, HttpError>;
+
+    /// Get the current oximeter read policy
+    #[endpoint {
+            method = GET,
+            path = "/oximeter-read/policy"
```

Comment:
I don't think that `oximeter-read` really means anything without the word policy. I'd make it all one path like `oximeter-read-policy`. That way it's only a noun.

Reply:
yeah, that makes sense. I changed it to "/oximeter/read-policy" bc it seemed nicer though